src/api/ledger/balances.js

```diff
@@ -31,14 +34,26 @@ function getTrustlinesAsync(account, options, callback) {
     .catch(callback);
 }
 
+function getLedgerVersion(remote: Remote, optionValue?: number,
+  callback: GetLedgerSequenceCallback
+) {
+  if (typeof optionValue === 'number') {
```

Comment:
I'd prefer `optionValue !== undefined && optionValue !== null` to avoid `typeof`.